airflow/secrets/local_filesystem.py

```diff
@@ -235,33 +235,33 @@ def load_variables(file_path: str) -> Dict[str, str]:
     return variables
 
 
-def load_connections(file_path: str):
+def load_connections(file_path: str) -> Dict[str, Any]:
     """
     Load connection from text file.
 
     Both ``JSON`` and ``.env`` files are supported.
 
     :return: A dictionary where the key contains a connection ID and the value contains a list of connections.
-    :rtype: Dict[str, List[airflow.models.connection.Connection]]
+    :rtype: Dict[str, airflow.models.connection.Connection]
```

Comment:
This is the most important part. This function does not need to return a list. because conn_id is unique.